core/src/main/scala/org/bykn/bosatsu/Evaluation.scala

```diff
@@ -276,10 +276,11 @@ object Evaluation {
 
 }
 
-case class Evaluation(pm: PackageMap.Inferred, externals: Externals) {
+case class Evaluation(inferredPm: PackageMap.Inferred, externals: Externals) {
   import Evaluation.{Value, Scoped, Env}
   import Value._
 
+  val pm = NormalizePackageMap(inferredPm).normalizePackageMap
```

Comment:
nit: if we require NormalizedPackages before Evaluation, I'd rather the constructor require it and then make an apply method on Evaluation to pass it in. This is a minor issue though.

Reply:
since Evaluation is now agnostic tags I'll normalize in `Main` and pass it in